core-plugins/src/main/java/co/cask/hydrator/plugin/transform/PythonEvaluator.java

```diff
@@ -344,6 +347,12 @@ private Object decodeUnion(Object object, List<Schema> schemas) {
   }
 
   private void init() {
+
+    // This is necessary to prevent 'ImportError' being thrown by Jython
+    Properties properties = new Properties();
+    properties.put("python.import.site", "false");
```

Comment:
@awholegunch It prevents the `site` module from being imported automatically.
@gsps1 I looked into the [2.7.0 release notes](https://wiki.python.org/jython/JythonReleaseNotes). Seems like site modules are only automatically imported beginning from version `2.7b3`.